example.js

```diff
@@ -0,0 +1,12 @@
+const runshell = require('./index.js');
+
+runshell('ls', {
+  args: {
+    _: ['-al']
```

Comment:
this is really confusing.  you are passing _ as dashed argument?.  Would be better to do 
`{ a: true, l: true }`